substack/01-backend-previews/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.22 as builder
+
+WORKDIR /
+COPY main.go .
+RUN go build -o goapp main.go
+
+FROM alpine:latest
+RUN apk add libc6-compat
```

Comment:
do we need this here? libc6-compat

Reply:
That's interesting, so I built your Dockerfile in docker-scout and it worked fine, I copied exactly your Dockerfile in my folder and built the code but when I try to run it said "goapp not found". Checked the FS and it's there, no permissions problem. 
Found out that alpine, gives you this error when something has been built with another version of the lib C than it does, because it uses musl instead of standard lib C. People said to add this dependency and it started to work ..